### PR TITLE
Update Authenticator interfaces

### PR DESCRIPTION
**Pull request by vanam**
Use [Nette Security 3.1](https://blog.nette.org/en/news-in-nette-security-3-1) interface.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		"contributte/application": "^0.5.0",
 		"contributte/di": "^0.5.0",
 		"contributte/cache": "^0.6.0",
-		"contributte/http": "^0.4.0",
+		"contributte/http": "0.5.x-dev",
```
Comment (vanam) [inline composer.json:21]: Waiting for https://github.com/contributte/http/commit/f1872fe39e04ea786ce79af9da333735a6c8ce44 to get into stable version.

Comment (f3l1x) [reply composer.json:21]: https://github.com/contributte/http/releases/tag/v0.4.1

**Comment by f3l1x** [comment]
It needs to be rebased.

**Comment by f3l1x** [comment]
👍

**Pull request merged**